GameServerLib/Logic/Chatbox/ChatCommandManager.cs

```diff
@@ -112,7 +112,7 @@ public void LoadCommands()
             AddCommand(new SetCommand("set", "set masterMask fieldMask", this));
             AddCommand(new SizeCommand("size", "size size", this));
             AddCommand(new SkillpointsCommand("skillpoints", "", this));
-            AddCommand(new SpawnCommand("spawn", "spawn minions", this));
+            AddCommand(new SpawnCommand("spawn", "spawn minionsblue, minionspurple", this));
```

Comment:
The command format here should be
`spawn [type] [variation]`

So in this case, it would be better to have
`spawn minions [blue|red|both]`
as the syntax